From 2b7137bfd464448bed3894492c29d6f806af6d89 Mon Sep 17 00:00:00 2001
From: Ryan Brady <rbrady@redhat.com>
Date: Thu, 3 Apr 2014 08:53:52 -0400
Subject: [PATCH] Move setup-clienttools to devtest_setup.sh

This patch moves the call to setup-clienttools from
install-dependencies to devtest_setup.sh to support a
near future patch to allow for using packages in
setup-clienttools.  If they're building images from
packages, they'll likely want to use packages on the
host as well.

Change-Id: I13bc19fb7d4a9fd347eb09ef0accc50f28ce0475
---
 scripts/devtest_setup.sh     | 1 +
 scripts/install-dependencies | 2 --
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/scripts/devtest_setup.sh b/scripts/devtest_setup.sh
index 958001f..774c640 100755
--- a/scripts/devtest_setup.sh
+++ b/scripts/devtest_setup.sh
@@ -198,6 +198,7 @@ fi
 
 if [ "$USE_CACHE" == "0" ] ; then #nodocs
     install-dependencies
+    setup-clienttools
 fi #nodocs
 
 ## #. (Optional) Run cleanup-env to delete VM's and storage pools from previous
diff --git a/scripts/install-dependencies b/scripts/install-dependencies
index 5a34089..2efd9f2 100755
--- a/scripts/install-dependencies
+++ b/scripts/install-dependencies
@@ -88,8 +88,6 @@ if [ "$os" = "suse" ]; then
   sudo service openvswitch-switch restart
 fi
 
-setup-clienttools
-
 # libvirtd group
 case "$os" in
     'Debian' | 'suse')
-- 
1.9.0

